js/redirect.js

```diff
@@ -0,0 +1,27 @@
+var matches = /Firefox\/(\d+)/.exec(navigator.userAgent);
+	
+// if using Fx
+if (matches !== null && matches.length > 1) {
+	var version = parseInt(matches[1], 10);
+
+	// if using a version with about:home (4 or greater), redirect them there
+	if(version >= 4) {
+		document.location.href = 'about:home';
+	}
+
+	// make sure platform is upgradeable (not on PPC or OS X 10.0 - 10.4)
+	if ( (/(PPC|Mac OS X 10.[0-4])/.test(navigator.userAgent) === false) && (version < 29) ) {
+		// display appropriate message
+		if (version === 3) {
+			$('#fx3').show();
+		} else if (version === 4) {
+				$('#fx4').show();
+		} else {
+				$('#fxother').show();
+		}
+	
+	}
```

Comment:
That makes sense. Slipped my mind that the pop-up logic was still handled within this code block. I'll update the PR.